OpenRA.Mods.Common/Activities/Turn.cs

```diff
@@ -9,7 +9,10 @@
  */
 #endregion
 
+using System.Collections.Generic;
+using System.Drawing;
 using OpenRA.Activities;
+using OpenRA.Mods.Common.Traits;
 using OpenRA.Traits;
```

Comment:
please revert the using changes to this class as you aren't doing anything here.